_data/conf.yml

```diff
@@ -94,5 +94,5 @@ livestream:
   url: "https://www.youtube.com/user/code4lib"
 
 # If true, any "Sponsors" links with go to /prospectus instead of /sponsors
-call-for-sponsors: true
+call-for-sponsors: false
```

Comment:
Changing this to false also removes the "Sponsor Code4Lib" button on the main page. We talked about changing this flag to switch from the prospectus to the actual sponsors during out meeting but is the sponsorship comm. ready to remove this call for sponsors from the main page since we don't have a lot of sponsors yet?

Reply:
I agree that we don't want to remove the call for sponsors link from the homepage.  Perhaps we could leave it in place but link to the sponsor page, which still links to the prospectus, just like the main menu